src/components/Hero/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Col, Container, Row } from 'react-bootstrap';
+import { SITE_NAME } from '@/constants';
+
+export function Hero() {
+    return (
+        <section className="hero py-5 mb-5">
+            <Container className="my-5 py-5">
+                <Row>
+                    <Col lg={6}>
+                        <p className="hero-text text-sm me-lg-5 pe-lg-5 mb-5 mb-lg-0">
+                            An MBA is predominately a networking or career
```

Comment:
- Bold `contrarian MBA`. Make it bright white, bold. Make it pop.
- Not sure I like the site name repeated on the right if it's already at the top
- Change the sentence which starts with `The curriculum is weak` to `Across the board, the curriculum is weak`
- Can you try to make the text left-align/word wrap; don't justify-content because the spacing feels less natural

Reply:
Right side site name removed `/` like color swap is removed. Used a darker green color.

Reply:
Updated screenshot